gdcdictionary/schemas/annotation.yaml

```diff
@@ -166,6 +166,12 @@ links:
         target_type: gene_expression
         multiplicity: many_to_many
         required: false
+      - name: masked_methylation_arrays
```

Comment:
also need to add these new links to `properties`

Reply:
done!